source/2020-06-23-ember-3-19-released.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Ember 3.19 Released
+author: Kenneth Larsen
+tags: Releases, 2020, 3, 3.19, Version 3.x
+responsive: true
+---
+
+Today the Ember project is releasing version 3.19 of Ember.js, Ember Data, and Ember CLI. This release kicks off the 3.20 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/releases/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.19
+
+Ember.js 3.19 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations. There is zero (0) new feature, zero (0) deprecations, and several bugfixes in this version.
+
+#### New Features (0)
+
+There are no new features introduced in Ember.js 3.19.
+
+#### Deprecations (0)
+
+There are no new deprecations introduced in Ember.js 3.19.
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 3.19
+
+#### New Features (0)
+
+There are no new features introduced in Ember Data 3.19.
+
+#### Deprecations (0)
```

Comment:
From the changelog, it looks there is 1 deprecation.

https://github.com/emberjs/data/blob/master/CHANGELOG.md#release-3190-june-5-2020